test/DynamoCoreWpfTests/ViewExtensions/NodeManipulatorExtensionTests.cs

```diff
@@ -28,7 +28,7 @@ protected override IEnumerable<IGizmo> GetGizmos(bool createOrUpdate)
     public class NodeManipulatorExtensionTests : DynamoTestUIBase
     {
         [SetUp]
-        public virtual void Start()
+        public override void Start()
```

Comment:
nice!. DId you try to turn on the dispatrcher ops waiting after this change ?

Reply:
I did, will give it a shot, were you able to repro the crash locally?

Reply:
on second thought, there's no reason for this method to exist if we're flushing the dispatcher.

Also, I think theres something weird with the way we increment the dispatcher ops when this flag is true...

Reply:
I did manage to reproduce the crash locally